plugin/someteam.example.com/v1/bashedconfigmap/go.mod

```diff
@@ -1,9 +1,41 @@
 module sigs.k8s.io/kustomize/plugin/someteam.example.com/v1/bashedconfigmap
 
-go 1.16
+go 1.18
 
 require sigs.k8s.io/kustomize/api v0.8.9
 
+require (
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
+	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
+	github.com/go-errors/errors v1.0.1 // indirect
+	github.com/go-openapi/jsonpointer v0.19.5 // indirect
+	github.com/go-openapi/jsonreference v0.19.5 // indirect
+	github.com/go-openapi/swag v0.19.14 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/google/gnostic v0.5.7-v3refs // indirect
+	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510 // indirect
+	github.com/imdario/mergo v0.3.6 // indirect
+	github.com/josharian/intern v1.0.0 // indirect
+	github.com/mailru/easyjson v0.7.6 // indirect
+	github.com/monochromegane/go-gitignore v0.0.0-20200626010858-205db1a8cc00 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/testify v1.7.0 // indirect
+	github.com/xlab/treeprint v1.1.0 // indirect
+	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
+	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd // indirect
+	golang.org/x/sys v0.0.0-20220209214540-3681064d5158 // indirect
+	golang.org/x/text v0.3.7 // indirect
+	google.golang.org/protobuf v1.28.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	k8s.io/kube-openapi v0.0.0-20220401212409-b28bf2818661 // indirect
+	sigs.k8s.io/kustomize/kyaml v0.13.7 // indirect
+	sigs.k8s.io/yaml v1.2.0 // indirect
+)
+
 replace sigs.k8s.io/kustomize/api => ../../../../api
 
 replace sigs.k8s.io/kustomize/kyaml => ../../../../kyaml
```

Comment:
With the new go workspace setup, I don't think we need to have these replace statements anymore. (Here, and in all the other plugins)

Reply:
I remove all `replace`, and cleanup to `go.mod` and `go.sum` files at https://github.com/kubernetes-sigs/kustomize/pull/4692/commits/0106c22e41085736e348b75ab61475b6d819b9a8